infrastructure/scripts/create-testdata/create-release.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 set -eu
 set -o pipefail
-#set -x
+set -x
 
 # usage
 # ./create-release.sh my-service-name [my-team-name]
 # Note that this just creates files, it doesn't push in git
 
 name=${1}
 applicationOwnerTeam=${2:-sreteam}
-commit_id=$(LC_CTYPE=C tr -dc a-f0-9 </dev/urandom | head -c 12 ; echo '')
+commit_id=$(echo $RANDOM)
```

Comment:
Can you make this look more look more like a git commit id? 12 hex characters are much better than just a number.

Reply:
this was just for testing with sven, removing this cahnge